x/staging/posts/client/cli/tx.go

```diff
@@ -351,13 +351,13 @@ E.g.
 // GetCmdReportPost returns the command allowing to report a post
 func GetCmdReportPost() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "report [post-id] [reports-type] [reports-message]",
+		Use:   "report [post-id] [reports-message] [reports-types...]",
```

Comment:
I don't link this because it makes it mandatory to set a report message, which should not. I think we can use commands flags to set the report reasons making the second parameter (the message) optional instead